tests/src/rules/no-extraneous-dependencies.js

```diff
@@ -243,24 +248,23 @@ ruleTester.run('no-extraneous-dependencies', rule, {
       options: [{packageDir: packageDirMonoRepoWithNested}],
       errors: [{
         ruleId: 'no-extraneous-dependencies',
-        message: "'left-pad' should be listed in the project's dependencies. Run 'npm i -S left-pad' to add it",
+        message: '\'left-pad\' should be listed in the project\'s dependencies. Run \'npm i -S left-pad\' to add it',
```

Comment:
are all these quotes changes necessary?

Reply:
yea.. shouldn't it prefer backticks?

Reply:
Only if there's interpolations.

Reply:
This still seems like it should be reverted.

Reply:
https://github.com/hulkish/eslint-plugin-import/blob/2e41085228ddd026ac7735eff36247b7f4d53d61/tests/src/rules/no-extraneous-dependencies.js as u can see, this change is consistent with the rest of the file. It actually fails lint for me with the existing (in master).

Reply:
actually, i can see now  this was corrected in master, so ill revert.